public/index.html

```diff
@@ -22,6 +22,23 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Arronax by Cryptonomic</title>
+    
+    <!-- Matomo -->
+  <script type="text/javascript">
```

Comment:
just a suggestion, create a file analytics.js in public, paste content and attach this way 
`<script src="%PUBLIC_URL%/analytics.js"></script>`